packages/react/src/components/__tests__/SignInButton.test.tsx

```diff
@@ -52,7 +52,7 @@ describe('<SignInButton/>', () => {
     const btn = screen.getByText('Sign in');
     await userEvent.click(btn);
 
-    expect(mockRedirectToSignIn).toHaveBeenCalledWith({ forceRedirectUrl: url });
+    expect(mockRedirectToSignIn).toHaveBeenCalledWith({ signInForceRedirectUrl: url });
   });
 
   it('handles forceRedirectUrl prop', async () => {
```

Comment:
Why do we have the same test twice?

Reply:
Good catch, no idea. Looks like the test was simplified after recent changes to the point where its the same with the one above